### PR TITLE
docs(snap): improve descriptions of config variables in snap/README.md

### PR DESCRIPTION
**Pull request by MonicaisHer**
Signed-off-by: Mengyi <mengyi.wang@canonical.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Comment by MonicaisHer** [comment]
Could you please review it? Thank you! @farshidtz

File: snap/README.md [hunk 1]
```diff
@@ -453,15 +453,15 @@ Example: `snap set edgexfoundry env.device-virtual.service.port=7777`
...
 | default-token-ttl      | The default-token-ttl setting is a Go Duration string, a sequence of decimal numbers, each with optional fraction and a unit suffix (e.g. "ns", "us" (or "µs"), "ms", "s", "m", "h"). It's used to set the TTL of vault tokens generated for EdgeX services during bootstrap. This setting can be used to increase (or decrease) the default TTL (one hour). If the TTL of a token expires before a service is started, the service will not be able to access the Secret Store. |
 
 ### Security bootstrapper settings (prefix: `env.security-bootstrapper.`)
 
 | setting                | Description                                                  |
 | ---------------------- | ------------------------------------------------------------ |
-| add-registry-acl-roles | Defines a JSON list of service keys used to create ACL roles in Vault to allow secure Consul access for the services. It contains a comma-separated list of registry role names to be added |
+| add-registry-acl-roles | Defines a csv list of service keys used to create ACL roles in Vault to allow secure Consul access for the services. It contains a comma-separated list of registry role names to be added |
```
Comment (farshidtz) [inline snap/README.md:464]: I'd call it "comma-separated list". Also, the second sentence is redundant.

File: snap/README.md [hunk 2]
```diff
@@ -453,15 +453,15 @@ Example: `snap set edgexfoundry env.device-virtual.service.port=7777`
...
+| add-secretstore-tokens | Defines a csv list of service keys to be added to the list of Vault tokens that security-file-token-provider (launched by security-secretstore-setup) creates. It is set to a default list of additional services by the snap, so be sure to examine the default setting before providing a custom list of services. NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-secretstore-setup. |
+| add-known-secrets      | The add-known-secrets setting is a csv list of secret paths and associated services. It's used to provision the specified secret for the given service in Vault. It is set to a default list of additional services by the snap, so be sure to examine the default setting before providing a custom list of services. NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-secretstore-setup. |
```
Comment (farshidtz) [inline snap/README.md:457]: Please change to "comma-separated list".

**Comment by farshidtz** [comment]
The description of `add-proxy-route` needs improvement too.

File: snap/README.md [hunk 3]
```diff
@@ -447,21 +447,21 @@ Example: `snap set edgexfoundry env.device-virtual.service.port=7777`
 
 | API Gateway setting | Description                                                  |
 | ------------------- | ------------------------------------------------------------ |
-| add-proxy-route     | Defines a configuration TOML containing a list of URLs to be added to the API Gateway (aka Kong). See [documentation](https://docs.edgexfoundry.org/2.1/security/Ch-APIGateway/#configuration-of-adding-microservices-routes-for-api-gateway) NOTE - this setting is not a configuration override, it's a top-level environment variable used by the security-proxy-setup. That means it can not be configureed thourgh `snap set` |
+| add-proxy-route     | Defines a configuration TOML file containing a comma-separated list of paired additional service names and URLs to create proxy Kong routes. See [documentation](https://docs.edgexfoundry.org/2.1/security/Ch-APIGateway/#configuration-of-adding-microservices-routes-for-api-gateway) NOTE - this setting is not a configuration override, which means it can not be configured through `snap set`. it's a top-level environment variable used by the security-proxy-setup. |
```
Comment (farshidtz) [inline snap/README.md:450]: I'm not sure why this is called a "configuration TOML file". From the linked document, this is a URL: "myApp.http://my-app:56789" and multiple URLs can be passed as a comma-separated list.

File: snap/README.md (hunk 1 above)
Comment (MonicaisHer) [reply snap/README.md]: Updated. Thanks!

File: snap/README.md (hunk 2 above)
Comment (MonicaisHer) [reply snap/README.md]: Updated.

File: snap/README.md (hunk 3 above)
Comment (MonicaisHer) [reply snap/README.md]: It was not right. I have updated it. Thanks!

**Review by siggiskulason — APPROVED** [review]
LGTM

**Review by farshidtz — APPROVED** [review]

**Comment by farshidtz** [comment]
Thanks @MonicaisHer. Merged.

**Pull request merged**